cmd/contour/contour.go

```diff
@@ -29,6 +31,9 @@ func main() {
 	log := logrus.StandardLogger()
 	k8s.InitLogging(k8s.LogWriterOption(log.WithField("context", "kubernetes")))
 
+	// set GOMAXPROCS
+	_, _ = maxprocs.Set(maxprocs.Logger(log.Printf))
```

Comment:
i think we do want be defensive and handle the error here, its unlikely to occur but good to still check

something like
```
if err != nil {
  log.WithError(err).Fatal("failed to set GOMAXPROCS")
}
```